[2025-05-21T09:32:34.127+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-21T09:32:34.166+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: merge_sources_pipeline.preprocessing manual__2025-05-21T09:20:57.597132+00:00 [queued]>
[2025-05-21T09:32:34.189+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: merge_sources_pipeline.preprocessing manual__2025-05-21T09:20:57.597132+00:00 [queued]>
[2025-05-21T09:32:34.190+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 3
[2025-05-21T09:32:34.218+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): preprocessing> on 2025-05-21 09:20:57.597132+00:00
[2025-05-21T09:32:34.250+0000] {standard_task_runner.py:72} INFO - Started process 956 to run task
[2025-05-21T09:32:34.272+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'merge_sources_pipeline', 'preprocessing', 'manual__2025-05-21T09:20:57.597132+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/inference_dag.py', '--cfg-path', '/tmp/tmpxo8ybru_']
[2025-05-21T09:32:34.280+0000] {standard_task_runner.py:105} INFO - Job 101: Subtask preprocessing
[2025-05-21T09:32:34.460+0000] {task_command.py:467} INFO - Running <TaskInstance: merge_sources_pipeline.preprocessing manual__2025-05-21T09:20:57.597132+00:00 [running]> on host c735ccec6410
[2025-05-21T09:32:34.702+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='merge_sources_pipeline' AIRFLOW_CTX_TASK_ID='preprocessing' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T09:20:57.597132+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-21T09:20:57.597132+00:00'
[2025-05-21T09:32:34.707+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-21T09:32:34.749+0000] {warnings.py:109} WARNING - /opt/***/scripts/preprocess.py:41: UserWarning: Could not infer format, so each element will be parsed individually, falling back to `dateutil`. To ensure parsing is consistent and as-expected, please specify a format.
  df["Take-off Time (UTC)"] = pd.to_datetime(df["Take-off Time (UTC)"], errors='coerce')

[2025-05-21T09:32:34.786+0000] {python.py:240} INFO - Done. Returned value was: []
[2025-05-21T09:32:34.838+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-21T09:32:34.839+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=merge_sources_pipeline, task_id=preprocessing, run_id=manual__2025-05-21T09:20:57.597132+00:00, execution_date=20250521T092057, start_date=20250521T093234, end_date=20250521T093234
[2025-05-21T09:32:34.927+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-21T09:32:34.979+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-21T09:32:34.983+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
